app/views/competitions/new.haml

```diff
@@ -32,6 +32,9 @@
     .form-group
       = f.label :mode, 'Mode'
       = f.select :mode, options_for_select(Competition.modes, :ffa), { include_blank: false }, class: "form-control"
+    .form-group
+      = f.label :secret_key, 'Secret Key'
+      = = f.text_field  :secret_key, 'form-control'
```

Comment:
Extra `= ` here. Haml is finicky.

Reply:
Whoops, VSCode auto-complete added an extra "=" in here. Fixed.